install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -103,3 +103,23 @@ spec:
             severity: warning
           annotations:
             message: "Memory usage of {{ $value | humanize }} on {{ $labels.node }} exceeds 90%. Master nodes starved of memory could result in degraded performance of the control plane."
+    - name: master-nodes-info-memory-usage
+      rules:
+        - alert: MasterNodesInfoMemoryUsage
+          expr: |
+            ((sum(node_memory_MemTotal_bytes AND on (instance) label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )) - sum(node_memory_MemFree_bytes + node_memory_Buffers_bytes + node_memory_Cached_bytes AND on (instance) label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" ))) / sum(node_memory_MemTotal_bytes AND on (instance) label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )) * 100) > 40
```

Comment:
can we wrap this, like:

```
(
  (
    sum(
      node_memory_MemTotal_bytes
      AND on (instance)
      label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
    ) - sum(
      node_memory_MemFree_bytes
      + node_memory_Buffers_bytes
      + node_memory_Cached_bytes
      AND on (instance)
      label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
    )
  ) / sum(
    node_memory_MemTotal_bytes
    AND on (instance)
    label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
  ) * 100
) > 40
```

or some such?  Also, might be slightly cheaper to evaluate if we refactor (`(a - b)/a` to be `1 - b/a`, like:

```
(
  1
  -
  sum(
    node_memory_MemFree_bytes
    + node_memory_Buffers_bytes
    + node_memory_Cached_bytes
    AND on (instance)
    label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
  ) / sum(
    node_memory_MemTotal_bytes
    AND on (instance)
    label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
  )
) * 100 > 40
```